app/views/comments/_comment_form.html.erb

```diff
@@ -40,7 +40,7 @@
         <% end %>
 
         <% if current_user.pseuds.count > 1 %>
-          <h4 class="heading"><%= ts("Comment as") %> <%= f.collection_select :pseud_id, current_user.pseuds, :id, :name, {:selected => (comment.pseud ? comment.pseud.id.to_s : current_user.default_pseud.id.to_s)}, :id => "comment_pseud_id_for_#{commentable.id}", :title => ts("Choose Name") %>
+          <h4 class="heading"><%= ts("Comment as") %> <%= f.collection_select :pseud_id, current_user.pseuds, :id, :name, {:selected: (comment.pseud ? comment.pseud.id.to_s  current_user.default_pseud.id.to_s)}, id: "comment_pseud_id_for_#{commentable.id}", title: ts("Choose Name") %>
```

Comment:
The missing : here appears to be what's causing the test failures: `comment.pseud ? comment.pseud.id.to_s  current_user.default_pseud.id.to_s`